challenge3/webapp/caesaris/app/lib/util.rb

```diff
@@ -0,0 +1,51 @@
+require "json"
+
+class Util
+  class << self
+    def form_ransack_params(params)
+      param_org = params
```

Comment:
本来やりたいのは、
```suggestion
      param_org = params.deep_dup
```
ではないだろうか。